tsconfig.json

```diff
@@ -0,0 +1,52 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "esnext",
+    "module": "commonjs",
+    "lib": ["es6"],
+    "allowJs": true,
+    "jsx": "react-native",
+    "types": ["react", "react-native"],
+    "declaration": true,
+    "sourceMap": true,
+    "isolatedModules": true,
+    "noEmitOnError": true,
+
+    /* Strict Type-Checking Options */
+    "strict": true,
```

Comment:
Because `strict` is true, you can remove some of the options below as they will become true by default. Looking at https://www.typescriptlang.org/tsconfig#strict and searching for `unless strict is set` should give you a good idea. Not essential though and some people do confirm being more explicit around these options, your call!